packages/bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js

```diff
@@ -41,6 +52,9 @@ const BpkHorizontalNavItem = props => {
   if (className) {
     innerClassNames.push(className);
   }
+  if (stretch) {
```

Comment:
The userland `className` addition is typically added last, so could you move the `if (stretch)` logic to go just above it?